cli/inspector.rs

```diff
@@ -0,0 +1,204 @@
+use crate::version::DENO;
+use futures::sync::oneshot::{spawn, SpawnHandle};
+use futures::{Future, Sink, Stream};
+use serde_json::Value;
+use std::net::SocketAddrV4;
+use std::sync::mpsc::{channel, Receiver, Sender};
+use std::sync::{Arc, Mutex};
+use warp::ws::{Message, WebSocket};
+use warp::Filter;
+
+static UUID: &str = "97690037-256e-4e27-add0-915ca5421e2f";
+// TODO(mtharrison): Make these configurable with flags, defaulting to below values
+static HOST: &str = "127.0.0.1";
+static PORT: &str = "9888";
+
+lazy_static! {
+  #[derive(Serialize)]
+  pub static ref RESPONSE_JSON: Value = json!([{
+    "description": "deno",
+    "devtoolsFrontendUrl": format!("chrome-devtools://devtools/bundled/js_app.html?experiments=true&v8only=true&ws={}:{}/websocket", HOST, PORT),
+    "devtoolsFrontendUrlCompat": format!("chrome-devtools://devtools/bundled/inspector.html?experiments=true&v8only=true&ws={}:{}/websocket", HOST, PORT),
+    "faviconUrl": "https://www.deno-play.app/images/deno.svg",
+    "id": UUID,
+    "title": format!("deno[{}]", std::process::id()),
+    "type": "deno",
+    "url": "file://",
```

Comment:
I am not familiar with Deno, but in Node some script files V8 reports have relative URLs. Some frontends use this URL to resolve those paths.